frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -59,10 +59,11 @@ function isBreakdownFunnelResults(results: FunnelStep[] | FunnelStep[][]): resul
     return Array.isArray(results) && (results.length === 0 || Array.isArray(results[0]))
 }
 
+// breakdown parameter could be a string (property breakdown) or object/number (list of cohort ids)
 function isValidBreakdownParameter(
     breakdown: FunnelRequestParams['breakdown']
 ): breakdown is string | null | undefined {
-    return ['string', 'null', 'undefined'].includes(typeof breakdown)
+    return ['string', 'null', 'undefined', 'object', 'number'].includes(typeof breakdown)
```

Comment:
Suggestion: For list, use `Array.isArray` instead.

Reply:
Is a multi-element cohort breakdown valid?

Reply:
yeah breakdown is an array if you pass in multiple cohorts